e2e/live/end-to-end-test.sh

```diff
@@ -113,6 +113,16 @@ shift $((OPTIND-1))
 #  Helper functions
 ###########################################################################
 
+function downloadPreviousKpt {
```

Comment:
So once we release the next version of kpt that includes support for the inventory policy, would this test still work?

Reply:
We can pin the kpt version used in this test. The last released version is v0.37.1. WDYT?

Reply:
That would address the first part. But is there a good reason to have this test here rather than in cli-utils?

Reply:
The cli-utils tests the library functionality. It doesn't test a package applied by a previous version of  the cli `kpt` can be applied by the new version of the cli `kpt`. Later the cli-utils may only contain the library part. So the `kpt` repo is a good place for this test.

Reply:
ok, that makes sense. But I do think we should make sure we have tests in cli-utils as well to cover the situation where a set of resources was added to the cluster without the `owning-inventory` annotation and then later is updated in way where the annotation are added. We might already cover this, but we should verify.

Reply:
Just verified that there is some e2e test covered the case without the `owning-inventory` annotation. https://github.com/kubernetes-sigs/cli-utils/blob/master/test/e2e/inventory_policy_test.go#L88

Reply:
Changed the version to v0.37.1. PTAL